exercises/twelve-days/twelve-days.spec.js

```diff
@@ -0,0 +1,106 @@
+var TwelveDays = require('./twelve-days');
+
+describe('TwelveDays', function () {
+  var twelveDaysObject;
+
+  beforeEach(function () {
```

Comment:
I don't believe this is necessary.  You can just instantiate a single instance and use it everywhere.  In fact, I would suggest that you _should_ do so to discourage submissions that might mutate their internal state.

Reply:
Okay, so declaring twelveDaysObject as a global should be fine?

Reply:
You're never mutating anything in the object, so it would be fine.